buildprocess/configureWebpackForPlugins.js

```diff
@@ -48,10 +47,9 @@ function createPluginIconsRule() {
       packageNames[svgPath] = packageName;
       return isTerriaJsPlugin;
     },
-    loader: require.resolve("svg-sprite-loader"),
+    loader: require.resolve("terriajs/buildprocess/svgs/svg-sprite-loader.js"),
     options: {
-      esModule: false,
-      symbolId: (svgPath) => {
+      namespace: (svgPath) => {
```

Comment:
This `namespace` function is currently returning a new name for each icon in the plugin. It should instead just return the package name, something like: `namespace: (svgPath) => packageNames[svgPath] || "terriajs-plugin"`